lib/frontend_helpers/icon_components.rb

```diff
@@ -451,6 +451,12 @@ module IconComponents
     </svg>
   )
 
+  COGS = %(
+    <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="cogs" role="img" viewBox="0 0 640 512" class="svg-inline--fa fa-cogs fa-w-20 fa-2x">
```

Comment:
acuerdate de limpiar esto
```suggestion
    <svg viewBox="0 0 640 512" class="svg-inline">
```